Fix: slice up QKeySequences correctly for Mute/Unmute

SlySven opened the pull request:
#### Brief overview of PR changes/additions
Adds a character to split up the OS dependent string `"Alt+K"` at the `+`.

#### Motivation for adding to Mudlet
PR #7440 tried to improve the UI by showing a message about which keys to press to undo a Mute/Unmute action the first time one of those is performed. Unfortunately the choice of the empty string as the separator in `QString::split(...)` means that the text is split up on a per character basis - and then reassembled using `>+<` as a joiner between the split items so that the above example was shown as `A>+<l>+<t>+<+>+<K` instead of `Alt>+<K` as it will now be...

#### Other info (issues closed, discussion etc)
This was reported by **@bcross32** in the Mudlet Discord **#Accessibility** channel on 2024/12/27T23:18:
https://www.discord.com/channels/283581582550237184/677242749514416170/1322342815866294373

SlySven committed:
Fix: slice up QKeySequences correctly for Mute/Unmute
#### Brief overview of PR changes/additions
Adds a character to split up the OS dependent string `"Alt+K"` at the `+`.

#### Motivation for adding to Mudlet
PR #7440 tried to improve the UI by showing a message about which keys to
press to undo a Mute/Unmute action the first time one of those is
performed. Unfortunately the choice of the empty string as the separator
in `QString::split(...)` means that the text is split up on a per character
basis - and then reassembled using `>+<` so that the above example was
shown as `A>+<l>+<t>+<+>+<K` instead of `Alt>+<K` as it will now be...

#### Other info (issues closed, discussion etc)
This was reported by **@bcross32** in the Mudlet Discord **#Accessibility**
channel on 2024/12/27T23:18:
https://www.discord.com/channels/283581582550237184/677242749514416170/1322342815866294373

Signed-off-by: Stephen Lyons <slysven@virginmedia.com>

mpconley commented:
Thanks. We need to determine how this looks on Mac, too, with the Command character. Unfortunately since these builds don't get signed, that's not a simple task.

ZookaOnGit commented:
> Thanks. We need to determine how this looks on Mac, too, with the Command character. 

Any input to #7590 would be appreciated to please.

SlySven committed:
Merge branch 'development' into Fix_sliceUp_QKeySequence_sCorrectlyForMuteAndUnmute

SlySven committed:
Merge branch 'development' into Fix_sliceUp_QKeySequence_sCorrectlyForMuteAndUnmute

vadi2 reviewed (approved):
Works well!

![image](https://github.com/user-attachments/assets/b72fffa7-c346-4fe3-bf3c-cee53205757d)

SlySven commented:
Oh :poop: that *wasn't* the effect I was hoping for - on other OSes I was intending that it should come out as `<Alt>+<K>` and `<⌘>+<K>` on MacOS - as Mudlet (Qt) cannot (and neither can my browser) handle the U+20E3 {COMBINING ENCLOSING KEYCAP} combining keycap codepoint and it would only combine with a single character:
![image](https://github.com/user-attachments/assets/116fae57-f510-44c8-a3f9-8c28b8835342)

so the `<`...`>` is trying to delimit each "key".

SlySven committed:
Revise: abandon attempt to display Mute/Unmute shortcut separated into keys

Instead just quote the whole string; since a similar message was being
used by the "Compact Input Line" do the same thing (just quoting) the same
thing there as well.

Signed-off-by: Stephen Lyons <slysven@virginmedia.com>

SlySven committed:
Merge branch 'development' into Fix_sliceUp_QKeySequence_sCorrectlyForMuteAndUnmute

SlySven commented:
Okay, I've removed the attempt to be fancy and show the text with a `<`...`>` around the key and the modifier. It now just quotes the sequence. Since a similar display is also used for the compact input line the first time it is activated I've repeated the same formatting for that message as well in this screen shot I've redefined the Mute/Unmute All shortcut to use more modifiers {I did try just Meta+L for the compact input line only to discover that that is the OS/DE one for "immediately lock the Desktop".... :man_facepalming:}:
![image](https://github.com/user-attachments/assets/b3797a8f-3544-41c6-966d-696966819191)

vadi2 commented:
Commit ca9b3c2d:

![image](https://github.com/user-attachments/assets/0837afc4-00e7-4104-b966-2a6bc8997eba)

mpconley commented:
Looks like it works for Vadi. I can't open these builds on arm it give me a crash. I will check this out once it hits the PTB.
